reultchecker.py

```diff
@@ -45,6 +45,8 @@ def check_api():
     registration_no = "540020503522"
     params = {"registration_no": registration_no, "login_type": "result"}
 
+
+
     while True:
         try:
             response = requests.get(url, params=params)
```

Comment:
**Static Code Issue**: Probable use of requests call without timeout <br> **Fix**: Add a timeout to the requests.get call to prevent hanging indefinitely if the server does not respond. <br> **Code Suggestion**: 
 ```
 - response = requests.get(url, params=params)
 + response = requests.get(url, params=params, timeout=10)
 ```